assignments/number-functions/code.js

```diff
@@ -0,0 +1,30 @@
+const add = (a,b) => {
+  return a + b ;
+};
+const subtract = (a,b) => {
+  return a - b ;
+};
+const multiply = (a,b) => {
+  return a * b ;
+};
+const divide = (a,b) => {
+  return a / b ;
+};
+const mod = (a,b) => {
+  return a % b ;
+};
+const averageOfTwo = (a,b) => {
+  return (a+b)/2 ;
+};
+const averageOfThree = (a,b,c) => {
+  return (a+b+c)/3 ;
+};
+const distance = (a,b) => {
+  return Math.abs(a-b) ;
+};
+const manhattanDistance = (x1,y1,x2,y2) => {
+  return Math.abs(x1-x2)+Math.abs(y1-y2) ;
+};
+const euclideanDistance = (x1,y1,x2,y2) => {
+  return Math.sqrt((Math.abs(x1-x2) * Math.abs(x1-x2)) + (Math.abs(y1-y2) * Math.abs(y1-y2))) ;
```

Comment:
Just a mathematical note: you don't actually need the `Math.abs` here since squaring a number always results in a positive value. Also you could make this a bit more efficient and also easier to read by using the `**` exponentiation operator:

```suggestion
  return Math.sqrt((x1-x2) ** 2  + (y1-y2) ** 2) ;
```